addons/medical_statemachine/stringtable.xml

```diff
@@ -4,33 +4,43 @@
         <Container name="Settings">
             <Key ID="STR_ACE_Medical_statemachine_subCategory">
                 <English>States</English>
+                <German>Zustände</German>
             </Key>
             <Key ID="STR_ACE_Medical_statemachine_fatalInjuryCondition_DisplayName">
                 <English>Fatal Injury Player</English>
+                <German>Tötliche Spielerverletzungen</German>
```

Comment:
```suggestion
                <German>Tödliche Spielerverletzungen</German>
```

That spieler part seems missplaced. The english translation also doesn't sound correct.